app/models/slang.rb

```diff
@@ -5,5 +5,5 @@ class Slang < ApplicationRecord
 
   acts_as_taggable_on :tags, :dialects
 
-  validates :name, presence: true, uniqueness: true, allow_blank: false
+  validates :name, presence: true, allow_blank: false
```

Comment:
Please don't change this line.
Instead, please add a verification to each slang when scraping.
It could be done with below code:
`unless Slang.exists?(name: "the scarping name")
    xxxxxxxx
end`